packages/coding-standard/src/Rules/PreferredRawDataInTestDataProviderRule.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Symplify\CodingStandard\Rules;
+
+use Nette\Utils\Strings;
+use PhpParser\Node;
+use PhpParser\Node\Expr\Variable;
+use PhpParser\Node\Stmt\ClassMethod;
+use PhpParser\NodeFinder;
+use PHPStan\Analyser\Scope;
+use PHPStan\Rules\Rule;
+use PHPStan\Type\ThisType;
+
+/**
+ * @see \Symplify\CodingStandard\Tests\Rules\PreferredRawDataInTestDataProviderRule\PreferredRawDataInTestDataProviderRuleTest
+ */
+final class PreferredRawDataInTestDataProviderRule implements Rule
+{
+    /**
+     * @var string
+     */
+    public const ERROR_MESSAGE = "Use raw data in test's \"dataProvider\" method";
```

Comment:
Maybe more clear message would be

"Code configured at setUp() cannot be used in data provider. Move it to test() method"